python/gaas_server/gaas_handler.py

```diff
@@ -333,19 +533,25 @@ def get_graph_edge_dataframe_rows(self,
             property_keys
         )
 
+        print(f'asked for edge indices: {index_or_indices}')
```

Comment:
should probably drop this print statement or only print in some sort of verbose mode